js/profile-w3c-common.js

```diff
@@ -21,10 +21,9 @@ require.config({
   deps: ["deps/hyperhtml", "deps/url-search-params"],
 });
 
-define(
+require(
```

Comment:
I don't know if this will build or not? I can't recall why it had to be define here.

Reply:
Seems it builds ok, looking at the non-failing Travis tests, so ignore my comment above.